common/api/repoDetails.ts

```diff
@@ -0,0 +1,38 @@
+import { HttpStatusCode } from "axios";
+
+export async function getRepoInfo(packageName = "", packageVersion = "") {
+    const npmPackageInfo = await fetch(
+        `https://registry.npmjs.org/${packageName}/${packageVersion}`,
```

Comment:
This should be resolvable from the package.json of the package?

We already make a call to fetch that using `pacote`. 
Can possibly avoid another API call.

Reply:
As far as I remember some of these info are not available in the `package.json` that's why I went for an API call instead. But. I'll check once again.

Reply:
Removed this call entirely. Now using `pacote` for the same.